Test_Cases/IEA-15-240-RWT-UMaineSemi/DISCON-UMaineSemi.IN

```diff
@@ -103,6 +104,13 @@
 0.00000             ! Y_IPC_KP			- Yaw-by-IPC proportional controller gain Kp
 0.00000             ! Y_IPC_KI			- Yaw-by-IPC integral controller gain Ki
 
+!------- LIDAR-ASSISTED CONTROL -------------------------------------------
```

Comment:
This style isn't quite consistent with the other inputs.  Ideally, we'd have `FF_Mode` and `FF_*` for the input parameters.   We can worry about this later.

Reply:
I've updated it to have FF_Mode